From: Christian Kastner <ckk@kvr.at>
Date: Mon, 21 Oct 2019 21:28:36 +0100
Subject: Get rid of BLAS

Remove references to the DFSG-removed BLAS implementation originally bundled
with upstream.

Forwarded: not-needed
Last-Update: 2019-10-21
---
 Makefile | 15 +++++----------
 1 file changed, 5 insertions(+), 10 deletions(-)

diff --git a/Makefile b/Makefile
index 0534f2b..99fdece 100644
--- a/Makefile
+++ b/Makefile
@@ -1,25 +1,24 @@
 CXX ?= g++
 CC ?= gcc
 CFLAGS = -Wall -Wconversion -O3 -fPIC
-LIBS = blas/blas.a
+LIBS =
 SHVER = 3
 OS = $(shell uname)
-#LIBS = -lblas
 
 all: train predict
 
-lib: linear.o tron.o blas/blas.a
+lib: linear.o tron.o
 	if [ "$(OS)" = "Darwin" ]; then \
 		SHARED_LIB_FLAG="-dynamiclib -Wl,-install_name,liblinear.so.$(SHVER)"; \
 	else \
 		SHARED_LIB_FLAG="-shared -Wl,-soname,liblinear.so.$(SHVER)"; \
 	fi; \
-	$(CXX) $${SHARED_LIB_FLAG} linear.o tron.o blas/blas.a -o liblinear.so.$(SHVER)
+	$(CXX) $${SHARED_LIB_FLAG} linear.o tron.o -o liblinear.so.$(SHVER)
 
-train: tron.o linear.o train.c blas/blas.a
+train: tron.o linear.o train.c
 	$(CXX) $(CFLAGS) -o train train.c tron.o linear.o $(LIBS)
 
-predict: tron.o linear.o predict.c blas/blas.a
+predict: tron.o linear.o predict.c
 	$(CXX) $(CFLAGS) -o predict predict.c tron.o linear.o $(LIBS)
 
 tron.o: tron.cpp tron.h
@@ -28,10 +27,6 @@ tron.o: tron.cpp tron.h
 linear.o: linear.cpp linear.h
 	$(CXX) $(CFLAGS) -c -o linear.o linear.cpp
 
-blas/blas.a: blas/*.c blas/*.h
-	make -C blas OPTFLAGS='$(CFLAGS)' CC='$(CC)';
-
 clean:
-	make -C blas clean
 	make -C matlab clean
 	rm -f *~ tron.o linear.o train predict liblinear.so.$(SHVER)
